src/main.cpp

```diff
@@ -794,7 +796,7 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
 
     // Rather not work on nonstandard transactions (unless -testnet/-regtest)
     string reason;
-    if (fRequireStandard && !IsStandardTx(tx, reason))
+    if (!policy.ApproveTx(tx, reason))
```

Comment:
No, I don't mean my branches. I mean that policies written today should not break subtlety tomorrow.